app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,30 @@
+class MoviesController < ApplicationController
+  def index
+    # Refactor this disaster into a MovieDB Poro?
+    conn = Faraday.new(url: 'https://api.themoviedb.org', headers: { 'Accept': 'application/json' })
+
+    page1 = conn.get('/3/movie/top_rated') do |req|
+      req.params['api_key'] = ENV["MOVIE_DB_API_KEY_V3"]
+      req.params['language'] = 'en-US'
+      req.params['page'] = '1'
+    end
+
+    page2 = conn.get('/3/movie/top_rated') do |req|
+      req.params['api_key'] = ENV["MOVIE_DB_API_KEY_V3"]
+      req.params['language'] = 'en-US'
+      req.params['page'] = '2'
+    end
+    
+    json1 = JSON.parse(page1.body, symbolize_names: true)
+    json2 = JSON.parse(page2.body, symbolize_names: true)
+
+    @movies_pg1 = json1[:results]
+    @movies_pg2 = json2[:results]
```

Comment:
it doesn't look like these instance variables are referenced outside of this file, so the `@` can probably be removed, no?

Reply:
whoops, yeah, they were originally instance variables bc I planned on using them both before I realized I could combine them. Will fix for next push.